<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HFontSet</class>
 <widget class="QWidget" name="HFontSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>274</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色(C)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fontStyleList">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>58</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="curFontStyle">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>30</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fontUnderlineComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>237</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="curFontPointSize">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="fontPointSizeList">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>58</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="fontFamilyList">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>58</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>7</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字形(O)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>7</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字号(S)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>预览</string>
   </property>
   <widget class="QFrame" name="fontPreview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="fontColorComboBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>300</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>213</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>7</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字体(M)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="curFontFamily">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="moreColorBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
